resources/testdriver.js

```diff
@@ -1141,6 +1141,82 @@
          */
         run_bounce_tracking_mitigations: function (context = null) {
             return window.test_driver_internal.run_bounce_tracking_mitigations(context);
+        },
+
+        /**
+         * Creates a virtual pressure source.
+         *
+         * Matches the `Create virtual pressure source
+         * <https://w3c.github.io/compute-pressure/#create-virtual-pressure-source>`_
+         * WebDriver command.
+         *
+         * @param {String} source_type - A `virtual pressure source type
+         *                               <https://w3c.github.io/compute-pressure/#dom-pressuresource>`_
+         *                               such as "cpu".
+         * @param {Object} [metadata={}] - Optional parameters described
+         *                                 in `Create virtual pressure source
+         *                                 <https://w3c.github.io/compute-pressure/#create-virtual-pressure-source>`_.
+         * @param {WindowProxy} [context=null] - Browsing context in which to
+         *                                       run the call, or null for the
+         *                                       current browsing context.
+         *
+         * @returns {Promise} Fulfilled when virtual pressure source is created.
+         *                    Rejected in case the WebDriver command errors out
+         *                    (including if a virtual pressure source of the
+         *                    same type already exists).
+         */
+        create_virtual_pressure_source: function(source_type, metadata={}, context=null) {
+            return window.test_driver_internal.create_virtual_pressure_source(source_type, metadata, context);
+        },
+
+        /**
+         * Causes a virtual pressure source to report a new reading.
+         *
+         * Matches the `Update virtual pressure source
+         * <https://w3c.github.io/compute-pressure/#update-virtual-pressure-source>`_
+         * WebDriver command.
+         *
+         * @param {String} source_type - A `virtual pressure source type
+         *                               <https://w3c.github.io/compute-pressure/#dom-pressuresource>`_
+         *                               such as "cpu".
+         * @param {String} sample - A `virtual pressure sample
```

Comment:
Looking at the spec, I think you meant "pressure state" instead of "pressure sample" here.

Reply:
Yes in this reference the text talking about the spec should be changed and is changed in this commit 7cc83255fb72c10e063aa47070051e35ee8834be

But the spec defines the parameter name as "`sample`" in _[Update virtual pressure source](https://www.w3.org/TR/compute-pressure/#update-virtual-pressure-source)_ -chapter. 

Also spec defines "`sample`" in sentence like _"A [pressure source](https://www.w3.org/TR/compute-pressure/#dfn-pressure-source)
has an associated latest sample_". 

And in one step of "Update virtual pressure source" service it says 
"_7. Let sample be the result of invoking get a property "sample" from parameters._"